cmd/drand-cli/cli.go

```diff
@@ -319,6 +319,12 @@ var appCommands = []*cli.Command{
 			return shareCmd(c)
 		},
 	},
+	{
+		Name:   "reload",
+		Usage:  "Launch a sharing protocol which has been previously stopped",
```

Comment:
This is confusing help text. Make it clear if this
* resuming resharing ceremony participation
* vs
* resuming participation in beacon generation for a named beacon.